.buildlibrary

```diff
@@ -11,6 +11,7 @@ AcceptedNotes:
 - Imports includes .* non-default packages.
 - unable to verify current time
 - checking installed package size
+- no visible global function definition for ':='
```

Comment:
Please rather add `@importFrom data.table :=` to `reportCrossVariables.R`, `plotNashConvergence.R` and `calc_CES_marginals.R` where it is currently missing.

And while you are at it, can you please adjust the unit of `Expenditure|Transport|Fuel` to `billion US$2005/yr` in [this line](https://github.com/pik-piam/remind/blob/master/R/reportCrossVariables.R#L239) such that we can close https://github.com/pik-piam/edgeTransport/issues/268?

Reply:
Sure! I did not know, that you can use it with dplyr as well..